demos/demos_by_use_case/bio/SingleCellDemo.ipynb

```diff
@@ -0,0 +1,935 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "5n26EioarG4C"
+      },
+      "source": [
+        "## Tutorial: Single-Cell Transcriptomics using UMAP"
+      ],
+      "id": "5n26EioarG4C"
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0f3a459c-fa15-445b-8d65-cb701c063ac9"
+      },
+      "source": [
+        "\n",
+        "Single cell gene expression can be analyzed faster and more easily explored using GPU-accelerated UMAP analysis & visualization. Using UMAP in this way, **the user can cluster cell types by patterns of gene expression**\n",
+        "\n",
+        "* Task: Analyze single cell's gene expression for clustering\n",
+        "* Data: 5 independent datasets of roughly 30K rows of 200 columns of single cell\n",
+        "* [data](https://cytotrace.stanford.edu/#shiny-tab-dataset_download)\n",
+        "* [paper](https://arxiv.org/pdf/2208.05229.pdf)\n",
+        "\n",
+        "**Insight/ Result:**\n",
+        "\n",
+        "1.   over 5x faster for entire ~10000 cell samples (102s vs 18s)\n",
+        "3.   Offers more insight when static plot would otherwise fail"
```

Comment:
```
        1.   Speed: Go from minutes to seconds for entire ~10000 cell samples (102s vs 18s)\n",
        2.   Visualization: Add interactivity, similarity edges, and GPU scale to otherwise hard-to-read static scatter plots"
```